### PR TITLE
fix(tests.openadapt.privacy.providers): failing `test_comprehend_scrub.py`

### PR DESCRIPTION
**Pull request by KrishPatel13**
**What kind of change does this PR introduce?**
Fixes pytest which is failing due to aws config files not found on machines.

Now, the tests in `test_comprehend_scrub.py` gets skipped if AWS Config files are not found or not correctly setup.

**Summary**

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have performed a self-review of my code
* [x] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

**How can your code be run and tested?**

**Other information**

### CODE REVIEW

**Comment by KrishPatel13** [comment]
This is how the pytest will look when:

1. There are no aws credentials setup/correctly for AWS Comprehend Test file. (See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration to setup)
2. There is no Spacy model (`en_core_web_trf`) installed for Presidio.

```
(openadapt-py3.10) PS P:\OpenAdapt AI - MLDS AI\cloned_repo\my_forked\OpenAdapt> pytest
============================================================================================ test session starts ============================================================================================
platform win32 -- Python 3.10.11, pytest-7.1.3, pluggy-1.3.0
rootdir: P:\OpenAdapt AI - MLDS AI\cloned_repo\my_forked\OpenAdapt, configfile: pyproject.toml
plugins: anyio-3.7.1
collected 43 items

tests\openadapt\test_crop.py .                                                                                                                                                                         [  2%]
tests\openadapt\test_events.py .......                                                                                                                                                                 [ 18%]
tests\openadapt\test_summary.py ..                                                                                                                                                                     [ 23%]
tests\openadapt\privacy\test_providers.py s                                                                                                                                                            [ 25%]
tests\openadapt\privacy\providers\test_comprehend_scrub.py sssssssssssssss                                                                                                                             [ 60%]
tests\openadapt\privacy\providers\test_presidio_scrub.py sssssssssssssssss                                                                                                                             [100%]

====================================================================================== 10 passed, 33 skipped in 8.20s =======================================================================================
(openadapt-py3.10) PS P:\OpenAdapt AI - MLDS AI\cloned_repo\my_forked\OpenAdapt> 
```

**Review by KrishPatel13 — COMMENTED** [review]
There are lot of unrelated changes from the alembic/versions directory. 

This is due to when I ran `pre-commit run -all` to fix the styling errors.

#self-reviewed

**Comment by KrishPatel13** [comment]
@abrichr Ready for merging!

**Comment by 0dm** [comment]
I believe if you run `poetry run black --preview` then it won't format the alembic files and still pass tests.

**Comment by KrishPatel13** [comment]
Yeah that is true.. But once we have them formatted then they won't interfere the `pre-commit run --all` as well :-)

**Review by Mustaballer — COMMENTED** [review]
I think you should ignore the changes to the migration files, because the files are generated.

**Comment by KrishPatel13** [comment]
> I think you should ignore the changes to the migration files, because the files are generated.

Shall I just delete all the version files. Is that fine ?

**Comment by Mustaballer** [comment]
@KrishPatel13 You shouldn't delete the migration files you should undo your changes to them

**Comment by FFFiend** [comment]
> @KrishPatel13 You shouldn't delete the migration files you should undo your changes to them

@Mustaballer Even if we ignore them, won't black formatting check fail because the check itself doesn't ? haha

**Comment by Mustaballer** [comment]
> > @KrishPatel13 You shouldn't delete the migration files you should undo your changes to them
> 
> @Mustaballer Even if we ignore them, won't black formatting check fail because the check itself doesn't ? haha

In the `pyproject.toml` the black setting ignores migration files, so when you run `poetry run black --check --preview .` for ex, it will not run anything on those migrations since its excluded.

**Comment by KrishPatel13** [comment]
@Mustaballer When I try to make commit and push the files,

The migration files get changed automatically.

What should I do now as It is not allowing me to push without the changes in migration files

File: alembic/versions/104d4a614d95_add_performancestat.py [hunk 1]
```diff
@@ -5,32 +5,33 @@
...
-from alembic import op
 import sqlalchemy as sa
 
+from alembic import op
 
 # revision identifiers, used by Alembic.
-revision = '104d4a614d95'
-down_revision = 'b2dc41850120'
+revision = "104d4a614d95"
+down_revision = "b2dc41850120"
```
Comment (abrichr) [inline alembic/versions/104d4a614d95_add_performancestat.py:14]: Black should be configured to ignore this directory

File: alembic/versions/104d4a614d95_add_performancestat.py [hunk 2]
```diff
@@ -5,32 +5,33 @@
...
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('performance_stat',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('recording_timestamp', sa.Integer(), nullable=True),
-    sa.Column('event_type', sa.String(), nullable=True),
-    sa.Column('start_time', sa.Integer(), nullable=True),
-    sa.Column('end_time', sa.Integer(), nullable=True),
-    sa.Column('window_id', sa.String(), nullable=True),
-    sa.PrimaryKeyConstraint('id', name=op.f('pk_performance_stat'))
+    op.create_table(
```
Comment (abrichr) [inline alembic/versions/104d4a614d95_add_performancestat.py:21]: Please remove unrelated changes

File: tests/openadapt/privacy/providers/test_comprehend_scrub.py [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
 
 from openadapt.privacy.providers.aws_comprehend import ComprehendScrubbingProvider
 
 scrub = ComprehendScrubbingProvider()
 
+try:
+    scrub.scrub_text("hello Bob smith")
+except NoRegionError:
+    pytestmark = pytest.mark.skip(
+        reason="AWS Config Files not setup correctly. Please see "
```
Comment (abrichr) [inline tests/openadapt/privacy/providers/test_comprehend_scrub.py:14]: Can you please define this message in a separate line? e.g.

```
msg = (
    "..."
    "..."
)
```

**Review by abrichr — DISMISSED** [review]

File: tests/openadapt/privacy/providers/test_comprehend_scrub.py (hunk 3 above)
Comment (KrishPatel13) [reply tests/openadapt/privacy/providers/test_comprehend_scrub.py]: Addressed in: 
aca3db5

File: alembic/versions/104d4a614d95_add_performancestat.py (hunk 2 above)
Comment (KrishPatel13) [reply alembic/versions/104d4a614d95_add_performancestat.py]: Addressed in: https://github.com/OpenAdaptAI/OpenAdapt/compare/71f37a4e2cc772cacec72c40ea14d7d15a54740e..1380d35e42f6651dc6f13fbc1dc652eee221f762

File: alembic/versions/104d4a614d95_add_performancestat.py (hunk 1 above)
Comment (KrishPatel13) [reply alembic/versions/104d4a614d95_add_performancestat.py]: Addressed in: https://github.com/OpenAdaptAI/OpenAdapt/compare/71f37a4e2cc772cacec72c40ea14d7d15a54740e..1380d35e42f6651dc6f13fbc1dc652eee221f762

**Comment by KrishPatel13** [comment]
@abrichr Ready for merging!

**Review by Mustaballer — APPROVED** [review]
